addons/storyshots/storyshots-core/src/frameworks/angular/renderTree.ts

```diff
@@ -13,7 +14,12 @@ function getRenderedTree(story: any) {
   const currentStory = story.render();
 
   const moduleMeta = getStorybookModuleMetadata(
-    { storyFnAngular: currentStory, parameters: story.parameters },
+    {
+      storyFnAngular: currentStory,
+      parameters: story.parameters,
+      // TODO : To change with the story Id in v7. Currently keep with static id to avoid changes in snapshots
```

Comment:
Should we add an issue and link as TODO? 
In my experience "TODO:" always get lost without an issue (that is taggable als 'do in v7' or something)

Reply:
I don't really know what to do. I don't use storyshots enough. So for me there is no real issue
Is it better to have shapshot with `storybook-wrapper` main div or with story id 🤷‍♂️ 

I can also change to a simple comment